src/main/java/org/folio/bulkops/service/DataExportJobUpdateService.java

```diff
@@ -124,14 +127,20 @@ private void downloadOriginFileAndUpdateBulkOperation(BulkOperation operation, J
       }
       operation.setEndTime(LocalDateTime.ofInstant(jobUpdate.getEndTime().toInstant(), UTC_ZONE));
 
+    } catch (S3ClientException e) {
```

Comment:
It makes to rework this also guiding by https://github.com/folio-org/mod-bulk-operations/pull/316/files#r1855540575 and https://github.com/folio-org/mod-bulk-operations/pull/316/files#r1855542832.